packages/ui/graphql/types/Report.ts

```diff
@@ -1,10 +1,18 @@
 import { extendType, objectType } from 'nexus';
+
 import { humanFileSize } from '../../lib/utils';
 import { getPackageID } from '../utils';
-
 import { MiniPackage, Package } from './Package';
 import { Suggestion } from './Suggestion';
 
+export const Top = objectType({
```

Comment:
Feels like this should be called `SuggestionCount` or `PackageCount` because that is what it is representing

Reply:
Updated